loader/CMakeLists.txt

```diff
@@ -315,6 +320,18 @@ else()
         add_library(vulkan STATIC ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS})
     else()
         add_library(vulkan SHARED ${NORMAL_LOADER_SRCS} ${OPT_LOADER_SRCS})
+        if(BUILD_WSI_XCB_SUPPORT)
+            find_package(XCB REQUIRED)
+            find_package(LibDRM REQUIRED)
+
+            target_include_directories(vulkan PRIVATE ${LibDRM_INCLUDE_DIRS} ${XCB_INCLUDE_DIRS})
+            target_link_libraries(vulkan ${LibDRM_LIBRARIES} ${XCB_LIBRARIES} xcb-dri3)
+        endif()
+#        if(BUILD_WSI_WAYLAND_SUPPORT)
```

Comment:
Commented out because why? Generally not good to have commented out code, but if there is a note about why its commented out (say its for a future change that isn't possible right now) its fine.

Reply:
This was removed.  The wayland code from Mesa relies on mesa-specific internals for now.  I'll have to figure out a different way to do this.  For now, the other options should be sufficient.